File: abc/407/c/b.c
#include <stdio.h>
#include <stdlib.h>

int main() {
    int x, y;
    
    // Ma'lumotlarni o'qish
    scanf("%d %d", &x, &y);
    
    int count = 0;
    
    // 36 xil holatni bittalab tekshirish
    for (int i = 1; i <= 6; ++i) {
        for (int j = 1; j <= 6; ++j) {
            // Mantiqiy operatorlar (||, >=) Python'dagi (or, >=) kabi ishlaydi
            if ((i + j >= x) || (abs(i - j) >= y)) {
                count++;
            }
        }
    }
    
    // Ehtimollikni hisoblash
    double probability = (double)count / 36.0;
    
    // Natijani 10 ta kasr xonasi aniqligida chop etish
    printf("%.10f\n", probability);
    
    return 0;
}

File: abc/407/py/b.py
def main():
    x, y = map(int, input().split())
    
    count = 0
    
    # Barcha 36 xil holatni tekshirib chiqamiz
    for i in range(1, 7):
        for j in range(1, 7):
            if (i + j >= x) or (abs(i - j) >= y):
                count += 1
                
    # Ehtimollikni hisoblash va chop etish
    probability = count / 36.0
    print(f"{probability:.10f}")

if __name__ == '__main__':
    main()